.travis.yml

```diff
@@ -10,7 +10,7 @@ matrix:
     - env: variant=test_debug sim=1 crypto=openssl toolset=gcc-sanitizer
     - env: variant=test_release coverage=1 tests=1 toolset=gcc-coverage python=1
     - env: autotools=1 toolset=gcc
-    - env: cmake=1 toolset=gcc
+    - env: cmake=1 toolset=gcc python=1
```

Comment:
this makes me a bit nervous. Don't you think it's enough to just always build python bindings when invoking cmake? we only do it in one configuration anyway

Reply:
I have no opinion on that. If your suggestion is to build the bindings always, that makes things easier, of course. Should I change that?

Reply:
I think the `cmake` invocation could always build the bindings, yes.

Reply:
OK, done.